exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillDatabaseMetaDataImpl.java

```diff
@@ -231,11 +274,16 @@ public boolean storesMixedCaseQuotedIdentifiers() throws SQLException {
     return super.storesMixedCaseQuotedIdentifiers();
   }
 
-  // TODO(DRILL-3510):  Update when Drill accepts standard SQL's double quote.
   @Override
   public String getIdentifierQuoteString() throws SQLException {
     throwIfClosed();
-    return "`";
+    Property property = getServerProperty(PlannerSettings.QUOTING_IDENTIFIERS_CHARACTER_KEY);
+    for (Quoting value : Quoting.values()) {
+      if (value.string.equals(property.getValue())) {
+        return value.string;
+      }
+    }
+    throw new SQLException("Unknown quoting identifier character " + property.getValue());
```

Comment:
Agree. I will add it after merging your's PRs (#613 and #764) to use logic from there.